ruby/ql/lib/codeql/ruby/Concepts.qll

```diff
@@ -484,6 +484,14 @@ module HTTP {
       /** Gets a node which returns the body of the response */
       DataFlow::Node getResponseBody() { result = super.getResponseBody() }
 
+      /**
+       * DEPRECATED: Use `getAUrlPart` instead.
```

Comment:
This isn't entirely related to this PR, but I was wondering what you thought about adding `(2022-03-24)` or similar to this comment so it's clear when the predicate was deprecated. I know we can estimate this using git blame, but that can become inaccurate if code gets moved around.

Even if it's a good idea, it might be something better added to all the deprecated predicates as a separate PR. I'm mostly just interested in thoughts.

Reply:
I think it's an excellent idea :+1: